backend/src/main/nl/tudelft/ti2806/riverrush/controller/UserController.java

```diff
@@ -44,7 +44,7 @@ public UserController(final EventDispatcher aDispatcher,
     public void initialize() {
         final HandlerLambda<JoinTeamCommand> joinTeamHandler = this::joinTeamHandler;
         final HandlerLambda<Event> sendOverNetworkLambda = (e) -> {
-            if (Objects.equals(e.getAnimal(), this.animal.getId())) {
+            if (Objects.equals(e.getAnimal(), this.animal.getId()) || Objects.equals(e.getAnimal(), -1)) {
```

Comment:
Add a check to see if the animal is in your team (because now you'll get BoatCollidedEvent for both boats, instead of just your own)